### PR TITLE
housekeeping: move to using Verify

### PR DESCRIPTION
**Pull request by SimonCropp**
So i was wading through the code, and noticed you use DiffEngine and also have a custom implementation of snapshot testing. So i figured i would throw up a PR that removes ApiExtensions.cs

### CODE REVIEW

**Comment by SimonCropp** [comment]
will fix the build today

**Comment by glennawatson** [comment]
No problem, thanks

**Comment by SimonCropp** [comment]
@glennawatson btw is the regex cleanup redundant? i tried removing it and the tests still pass? or is the `Coverlet` cleanup stuff not required in a local debug test run?

File: src/Splat.Drawing.Tests/API/ApiApprovalTests.cs [hunk 1]
```diff
@@ -4,23 +4,25 @@
 // See the LICENSE file in the project root for full license information.
 
 using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+using VerifyXunit;
 using Xunit;
 
+#pragma warning disable SA1615 // Element return value should be documented
```
Comment (SimonCropp) [inline src/Splat.Drawing.Tests/API/ApiApprovalTests.cs:11]: I would prefer this to be suppressed in test projects. let me know if you want a PR

**Comment by SimonCropp** [comment]
also this should be squashed on merge

**Comment by glennawatson** [comment]
I'll be holding off merging this PR probably to the end of the day. Gotta work out why the coverage reports aren't uploading in general.

**Comment by SimonCropp** [comment]
no worries on holding off. let me know if u want the regex removed as part of this pr

**Comment by glennawatson** [comment]
Go ahead and remove that regex. 

I think the codecov thing is likely just due to you being on a fork since it worked for another PR of mine.

**Comment by SimonCropp** [comment]
regex removed

**Review by glennawatson — APPROVED** [review]

**Pull request merged**